lib/compat/wordpress-6.1/class-wp-theme-json-6-1.php

```diff
@@ -1148,6 +1148,11 @@ protected static function get_property_value( $styles, $path, $theme_json = null
 	public function set_spacing_sizes() {
 		$spacing_scale = _wp_array_get( $this->theme_json, array( 'settings', 'spacing', 'spacingScale' ), array() );
 
+		// If theme authors want to prevent the generation of the core spacing scale they can set their theme.json spacingScale.steps to 0.
+		if ( 0 === $spacing_scale['steps'] ) {
```

Comment:
Oh, actually just another thought. This probably doesn't matter in practice because the base `theme.json` contains the `'steps'` key, but if we keep the `0` check, should this line and the `> 0` line below occur after the `! isset( $spacing_scale['steps'] )` check further down, to avoid an undefined index notice?

Reply:
yeh, makes more sense to check for invalid values first and then check for 0, have switched the order